ci-operator/config/openshift/ocp-release-operator-sdk/openshift-ocp-release-operator-sdk-master.yaml

```diff
@@ -7,52 +15,26 @@ base_images:
     name: builder
     namespace: ocp
     tag: rhel-9-golang-1.22-openshift-4.17
-  os:
-    name: builder
-    namespace: ocp
-    tag: rhel-8-base-openshift-4.9
-  os-minimal:
-    name: builder
-    namespace: ocp
-    tag: rhel-8-base-openshift-4.9
 build_root:
   from_repository: true
 images:
 - dockerfile_path: ci/dockerfiles/builder.Dockerfile
   from: root
   to: osdk-builder
-- dockerfile_path: release/ansible/Dockerfile.ocp
-  from: os
-  inputs:
-    ocp_builder_rhel-8-golang-1.22-openshift-4.17:
-      as:
-      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.22-openshift-4.17
-  to: ansible-operator
 - dockerfile_path: release/helm/Dockerfile
-  from: os-minimal
+  from: base
   inputs:
     ocp_builder_rhel-8-golang-1.22-openshift-4.17:
       as:
       - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.22-openshift-4.17
   to: helm-operator
 - dockerfile_path: release/sdk/Dockerfile
-  from: os-minimal
+  from: base-rhel9
```

Comment:
What are the reasons to move away from minimal base image to larger rhel 9?

Reply:
`os-minimal` base image is defined here: https://github.com/openshift/release/pull/54871/files#diff-9b461475076551ad0e6f09d15d820f229795658fba32acc0076ca868a7ec482dL14-L17

It is based on rhel8 and uses openshift 4.9. We shouldn't use openshift 4.9 as the base image for building any images in 4.17/4.18.

I have introduced the definition of `base` and `base-rhel9` here: https://github.com/openshift/release/pull/54871/files#diff-9b461475076551ad0e6f09d15d820f229795658fba32acc0076ca868a7ec482dR2-R9
These ocp version are updated using an automated process every release and we don't have to create PRs to keep this updated.

This are the reasons to move to the new definition of `base-rhel9`.

Reply:
Does ubi9 based images suffice for this?

Reply:
These are used as the base images for building the helm operator and operator-sdk images. As we use rhel base with latest OCP in the Dockerfiles ([operator-sdk](https://github.com/openshift/ocp-release-operator-sdk/blob/d147fd3b63cf4a6bc4c1d9168f5388f2da102505/release/sdk/Dockerfile#L10) and [helm-operator](https://github.com/openshift/ocp-release-operator-sdk/blob/d147fd3b63cf4a6bc4c1d9168f5388f2da102505/release/helm/Dockerfile#L11)), rhel should be used here.

Reply:
The difference in license between UBI and RHEL restricts the usage and re-distribution of container images being released with rhel base image. It is best to revisit from the customer perspective on the rhel vs ubi images for release.

Reply:
Approving based on the current understanding that CI images will be not re-distributed.